lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1182,6 +1183,8 @@ def _resource_sku_to_capability(
         node_space.name = f"{location}_{resource_sku.name}"
         node_space.features = search_space.SetSpace[str](is_allow_set=True)
         for sku_capability in resource_sku.capabilities:
+            if "standardLSv2Family" == resource_sku.family:
```

Comment:
write like below, it's extendable.
`if resource_sku.family in ["standardLSv2Family"]:`